css-brochure-form/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Learn More</h1>
+    <div>
+      <label for="CompanyEmail" id="email1">Company Email</label>
+    </div>
+    <div>
+      <input type="email" id="CompanyEmail" value="sally@company.co" />
```

Comment:
`id` values should be camelCased or kebab-cased. So they shouldn't start with a capital letter. They should always start with a lowercase letter.